dll/win32/mpr/mpr.spec

```diff
@@ -56,38 +58,51 @@
 @ stdcall WNetCancelConnectionA(str long)
 @ stdcall WNetCancelConnectionW(wstr long)
 @ stdcall WNetClearConnections(ptr)
-@ stdcall WNetCloseEnum(long)
+@ stdcall WNetCloseEnum(ptr)
 @ stdcall WNetConnectionDialog1A(ptr)
 @ stdcall WNetConnectionDialog1W(ptr)
 @ stdcall WNetConnectionDialog(long long)
+@ stub WNetConnectionDialog2 # (ptr long wstr long)
+@ stub WNetDirectoryNotifyA
+@ stub WNetDirectoryNotifyW # (ptr wstr long)
 @ stdcall WNetDisconnectDialog1A(ptr)
 @ stdcall WNetDisconnectDialog1W(ptr)
 @ stdcall WNetDisconnectDialog(long long)
+@ stub WNetDisconnectDialog2 # (ptr long wstr long)
 @ stdcall WNetEnumCachedPasswords(str long long ptr long)
-@ stdcall WNetEnumResourceA(long ptr ptr ptr)
-@ stdcall WNetEnumResourceW(long ptr ptr ptr)
+@ stdcall WNetEnumResourceA(ptr ptr ptr ptr)
+@ stdcall WNetEnumResourceW(ptr ptr ptr ptr)
 @ stub WNetFMXEditPerm
 @ stub WNetFMXGetPermCaps
 @ stub WNetFMXGetPermHelp
 @ stub WNetFormatNetworkNameA
-@ stub WNetFormatNetworkNameW
+@ stub WNetFormatNetworkNameW # (wstr wstr ptr ptr long long)
 @ stdcall WNetGetCachedPassword(ptr long ptr ptr long)
+@ stub WNetGetConnection2A
+@ stub WNetGetConnection2W # (wstr ptr ptr)
+@ stub WNetGetConnection3A
+@ stub WNetGetConnection3W
 @ stdcall WNetGetConnectionA(str ptr ptr)
 @ stdcall WNetGetConnectionW(wstr ptr ptr)
-@ stub WNetGetDirectoryTypeA
+@ stdcall -stub WNetGetDirectoryTypeA(str ptr long)
```

Comment:
Stub-enabling `WNetGetDirectoryTypeA()` was suggested, but does not seem required.
Tell me if you prefer not to.